reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -0,0 +1,652 @@
+PowerShell Core 6.x, PowerShell 7 is an in-place upgrade that removes PowerShell Core 6.x.
+
+- PowerShell 7 is installed to `%programfiles%\PowerShell\7`
+- The `%programfiles%\PowerShell\7` folder is added to `$env:PATH`
+
+The PowerShell 7 installer packages upgrade previous versions of PowerShell Core 6.x:
+
+- PowerShell Core 6.x on Windows: `%programfiles%\PowerShell\6` is replaced by
+  `%programfiles%\PowerShell\7`
+- Linux: `/opt/microsoft/powershell/6` is replaced by `/opt/microsoft/powershell/7`
+- macOS: `/usr/local/microsoft/powershell/6` is replaced by `/usr/local/microsoft/powershell/7`
+
+> [!NOTE]
+> In Windows PowerShell, the executable to launch PowerShell is named `powershell.exe`. In version 6
+> and above, the executable is changed to support side-by-side execution. The new executable to
+> launch PowerShell 7 is `pwsh.exe`. Preview builds will remain in-place as `pwsh-preview` instead
+> of `pwsh` under the 7-preview directory.
+
+## Improved shell experience with Predictive IntelliSense
+
+TODO
+
+## TODO - Add other Features
+
+## Breaking Changes and Improvements
+
+(TODO:IsComplete-ThruRC2)
+
+### Breaking Changes
+
+- **v7.1.0-preview.7**
+- Fix $? to not be $false when native command writes to stderr (#13395)
+- **v7.1.0-preview.6**
+- Rename -FromUnixTime to -UnixTimeSeconds on Get-Date to allow Unix time input (#13084) (Thanks @aetos382!)
+- Make $ErrorActionPreference not affect stderr output of native commands (#13361)
+- Allow explicitly specified named parameter to supersede the same one from hashtable splatting (#13162)
+- **v7.1.0-preview.4**
+- Make the switch parameter -Qualifier not positional for Split-Path (#12960) (Thanks @yecril71pl!)
+- Resolve the working directory as literal path for Start-Process when it's not specified (#11946) (Thanks @NoMoreFood!)
+- Make -OutFile parameter in web cmdlets to work like -LiteralPath (#11701) (Thanks @iSazonov!)
+- **v7.1.0-preview.3**
+- Fix string parameter binding for BigInteger numeric literals (#11634) (Thanks @vexx32!)
+- **v7.1.0-preview.2**
+- On Windows, Start-Process creates a process environment with all the environment variables from
+  current session, using -UseNewEnvironment creates a new default process environment (#10830)
+  (Thanks @iSazonov!)
+- Do not wrap return result to PSObject when converting ScriptBlock to delegate
+  (#10619)
+- **v7.1.0-preview.1**
+- Use invariant culture string conversion for -replace operator (#10954) (Thanks @iSazonov!)
+
+### Engine Updates and Fixes
+
+- **v7.1.0-preview.rc.2**
+- Rename Get-Subsystem to Get-PSSubsystem and fix two related minor issues (#13765)
+- Add missing PSToken token table entries to fix the PSParser API (#13779)
+- Add additional PowerShell modules to the tracked modules list (#12183)
+- Fix blocking wait when starting file associated with a Windows application (#13750)
+- Revert PSNativePSPathResolution to being an experimental feature (#13734)
```

Comment:
Remove this - since the feature was reverted it does not apply to GA. Also, remove the related item that added the feature being reverted.